Update isc-spring-2019-speakers.md (#217)
paypal/InnerSourceCommons: events/isc-spring-2019-speakers.md

</p>
</div>
<div style="clear:both;"></div>

<a name="Tobie_Langel">&nbsp;</a>
<hr/>
<div style="width:100%">
    <div style="float:left;padding-right:30px;padding-bottom:30px">
      <img alt="Tobie Langel" width="200" src="InnerSourceCommons/assets/img/Tobie_Langel.jpg"/>
    </div>
    <b>Tobie Langel</b></a>
    <p>
Tobie Langel is the founder of UnlockOpen, a boutique consulting firm that helps large organizations understand and leverage contributing to open source to recruit, retain, and foster top software engineering talent, and improve their teams’ efficiency, culture, and morale. His clients include top tech companies like Google, Microsoft, Intel, or Mozilla. Previously, he was a member of Facebook’s Open Source and Web Standards team, and was Facebook’s Advisory Committee representative at W3C. An avid open-source contributor, Tobie Langel is know for having co-maintained the Prototype JavaScript Framework and for numerous open source projects. He also edited a number of Web standards, including WebIDL, and led W3C’s Web platform testing effort.
  </p>
  </div>
<div style="clear:both;"></div>

> <a name="Johannes_Lenucksi">&nbsp;</a>

Reviewer: ```suggestion
<a name="Johannes_Tigges">&nbsp;</a>
```

<hr/>
<div style="width:100%">
    <div style="float:left;padding-right:30px;padding-bottom:30px">

>   <img alt="Johannes Lenucksi" width="200" src="InnerSourceCommons/assets/img/Picture Pending.png"/>

Reviewer: ```suggestion
  <img alt="Johannes Tigges" width="200" src="InnerSourceCommons/assets/img/Picture Pending.png"/>
```

Reviewer: I'll add a picture later here.

    </div>

>     <b>Johannes Lenucksi</b></a>

Reviewer: ```suggestion
    <b>Johannes Tigges</b></a>
```

    <p>

> Johannes is an Open Source enthusiast working for GitHub. He is helping large companies in the DACH and Eastern European region to build large internal software communities. Before GitHub, he was leading multiple Java development teams across the world building large Git backends and integration platforms.

Reviewer: ```suggestion
Johannes is an Open Source and Inner Source enthusiast currently working for HERE Technologies. He is working on establishing Inner Source and internal software communities within HERE. Before that he established and contributed to CI, CD and DevOps efforts in various other organizations - be it the technical or organizational and cultural aspects. He has a background in computer science and sociology and is generally interested in organizations, the human aspects of the software development domain as well as technical approaches to facilitating and measuring those.
```

Added a better description
